### PR TITLE
add control/status API

### PR DESCRIPTION
**Pull request by Avin0323**
### PR types

New features

### PR changes

APIs

### Describe

新增控制/状态类API：

- `paddle.is_floating_point`;
- `paddle.Tensor.is_floating_point`;
- `paddle.is_grad_enabled`;

中文文档：

image

image

image

英文文档：

image

image

image

### CODE REVIEW

File: python/paddle/framework/framework.py [hunk 1]
```diff
@@ -116,3 +116,24 @@ def set_grad_enabled(mode):
             tracer._has_grad = prev_mode
     else:
         yield
+
+
+def is_grad_enabled():
+    """
+    Returns currently dygraph gradient calculation mode which is enabled.
```
Comment (zhhsplendid) [inline python/paddle/framework/framework.py]: English: 
“Returns whether current dygraph gradient calculation mode is enabled”
Or "Returns whether dygraph gradient calculation mode is enabled currently"

1. "Current" is adj. "Currently" is adv. So you could say "current mode" but not "currently mode".
2. 用“Whether“表示是否。你上面的英语里which是修饰作用。所以你那句的英语意思是：”Return .... mode which is .... “, which修饰mode。你上面的英语意思是返回一个mode。

File: python/paddle/framework/framework.py [hunk 2]
```diff
@@ -116,3 +116,24 @@ def set_grad_enabled(mode):
...
+
+    Examples:
+        .. code-block:: python
+            
+            import paddle
+            
+            paddle.is_grad_enabled() # True
```
Comment (zhhsplendid) [inline python/paddle/framework/framework.py:134]: You could add comment "Dygraph gradient calculation mode is enabled by default."

File: python/paddle/tensor/__init__.py [hunk 3]
```diff
@@ -16,6 +16,7 @@
 from .attribute import shape  # noqa: F401
 from .attribute import real  # noqa: F401
 from .attribute import imag  # noqa: F401
+from .attribute import is_floating_point  # noqa: F401
```
Comment (zhhsplendid) [inline python/paddle/tensor/__init__.py:19]: I checked the code of `is_floating_point`, do we have to add API document for this?

File: python/paddle/framework/framework.py (hunk 1 above)
Comment (Avin0323) [reply python/paddle/framework/framework.py]: done

File: python/paddle/framework/framework.py (hunk 2 above)
Comment (Avin0323) [reply python/paddle/framework/framework.py:134]: done

File: python/paddle/tensor/__init__.py (hunk 3 above)
Comment (Avin0323) [reply python/paddle/tensor/__init__.py:19]: done

**Review by zhhsplendid — DISMISSED** [review]
LGTM

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LG API

**Pull request merged**